src/new-task-form/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react'
+import { useDispatch } from 'react-redux'
+
+import { onAddItem } from '../store/slice/todo.slice'
+import './new-task-form.css'
+import { REGEX, REGEX_2 } from '../common/constants'
+
+const NewTaskForm = () => {
+  const dispatch = useDispatch()
+
+  const [description, setDescription] = useState('')
+  const [min, setMin] = useState('')
+  const [sec, setSec] = useState('')
+
+  const onSubmitDescription = (e) => {
+    e.preventDefault()
+
+    setDescription(description.replace(REGEX, ' ').replace(REGEX_2, ''))
+
+    if (description.trim() !== '') {
+      dispatch(onAddItem({ description, min, sec }))
+
+      setDescription('')
+      setMin('')
+      setSec('')
+    }
+  }
```

Comment:
```suggestion
    if (!description.trim()) return 

      dispatch(onAddItem({ description, min, sec }))

      setDescription('')
      setMin('')
      setSec('')

  }
```


лучше так переписать